core/src/main/java/nu/yona/server/subscriptions/entities/User.java

```diff
@@ -25,6 +25,11 @@
 @Table(name = "USERS")
 public class User extends EntityWithID
 {
+	public enum Status
+	{
+		UNCONFIRMED, INACTIVE, ACTIVE
```

Comment:
INACTIVE and ACTIVE currently mean nothing. Please leave them out of the code for now until it is clear how we are going to use this. (Use boolean isConfirmed)